site/content/docs/main/how-velero-works.md

```diff
@@ -38,6 +38,8 @@ By default, backup storage locations are created in read-write mode. However, du
 
 You can optionally specify restore hooks to be executed during a restore or after resources are restored. For example, you might need to perform a custom database restore operation before the database application containers start. [More about restore hooks][11].
 
+During a restore, Velero is non-destructive. That means that Velero will never overwrite a pre-existing resource which has the same name as a resource in the backup it is restoring. Similarly, if a persistent volume exists, Velero will not overwrite the data in it. This behavior is by design, so that you don't have to worry about accidentally overwriting your cluster's state when using Velero.
```

Comment:
This is not exactly true for v1.9+

A contributor from redhat recently added an option so that some resources can be overwritten:
https://github.com/vmware-tanzu/velero/issues/4851#issuecomment-1108617077

Reply:
thanks for bringing this up, I'll close this PR and address this with the changes in PR https://github.com/vmware-tanzu/velero/pull/4928